server/controllers/middleware/index.js

```diff
@@ -0,0 +1,42 @@
+const { verify } = require('jsonwebtoken');
+require('env2')('config.env');
+
+const isClient = (req, res, next) => {
+  try {
```

Comment:
you have to be sure if there is a token before verify and send a bad request if not.